torchmultimodal/models/vqvae.py

```diff
@@ -52,18 +53,30 @@ def __init__(
         self.decoder = decoder
         self.codebook = Codebook(codebook_num_embeddings, codebook_embedding_dim)
 
-    def encode(self, x: Tensor) -> CodebookOutput:
-        return self.codebook(self.encoder(x))
+    def latent_shape(self, input_shape: Union[Size, Tuple]) -> Tuple[int, ...]:
+        return self.encoder.get_latent_shape(input_shape)  # type: ignore
+
+    def encode(
+        self, x: Tensor, return_embeddings: bool = False
+    ) -> Union[Tuple[Tensor, Tensor], Tensor]:
```

Comment:
add a quick comment here that this goes from input -> token_ids

Reply:
Added